test/Elastic.Apm.Tests/SystemInfoHelperTests.cs

```diff
@@ -9,6 +9,7 @@
 using Elastic.Apm.Helpers;
 using Elastic.Apm.Tests.Utilities;
 using FluentAssertions;
+using Newtonsoft.Json.Linq;
```

Comment:
```suggestion
```

and `using Elastic.Apm.Api;` can be removed as well (I can't change it here, because it's untouched)